### ISSUE TITLE
[FEATURE REQUEST] Open on current window

### ISSUE THREAD

**Issue by Altair-Bueno** [dwarvesf/session-buddy#15]
Allow the user to open saved tabs on the current Safari windows instead of creating a new window. 

### Expected behaviour

Settings allow the user to choose between opening a new window or opening on current window. Clicking the "open" button next to a session should behave as the selected preference. `option-click` on the open button should toggle the inverse behaviour.

**Comment by bluecursor** [comment]
Yes, I would like to have this feature as well to limit the number of windows opened
